app/src/main/java/com/kickstarter/libs/rx/transformers/Transformers.java

```diff
@@ -184,4 +185,11 @@ private Transformers() {}
   public static @NonNull <T> ObserveForUITransformer<T> observeForUI() {
     return new ObserveForUITransformer<>();
   }
+
+  /**
+   * Emits the second of any input pair
+   */
+  public static <R> R second(final @NonNull Pair<?, R> anyPair) {
```

Comment:
let's move this to a `PairUtils` classs and create the corresponding `first` cause I'm sure we will find that useful soon enough!